languages/ruby/lib/projects.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+class ProjectsClient
+  def initialize(command_runner)
+    @command_runner = command_runner
+  end
+
+  def create_project(project_name, organization_id)
+    @command_runner.run(
+      SelectiveCommand.new(
+        projects:
+          SelectiveProjectsCommand.new(
+            create: ProjectCreateRequest.new(
+              project_create_request_name: project_name,
+              organization_id: organization_id
+          )
+        )
+      )
+    )
+  end
+
+  def get(project_id)
+    @command_runner.run(
+      SelectiveCommand.new(
+        projects:
+          SelectiveProjectsCommand.new(
+            get: ProjectGetRequest.new(id: project_id)
+          )
+      )
+    )
```

Comment:
Please unwrap all the commands and handle the error, similar to the .NET sdk, https://github.com/bitwarden/sdk/blob/master/languages/csharp/Bitwarden.Sdk/ProjectsClient.cs#L12-L23